### PR TITLE
Fix CSP rules for Sentry client reporting

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons-frontend/issues/1933

### CODE REVIEW

File: config/default-admin.js [hunk 1]
```diff
@@ -13,7 +14,7 @@ module.exports = {
       styleSrc: [staticHost],
       imgSrc: [
         "'self'",
-        amoCDN,
+        amoProdCDN,
```
Comment (muffinresearch) [inline config/default-admin.js]: I like the name change - it's clearer.

File: config/default-admin.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
-const amoCDN = 'https://addons.cdn.mozilla.net';
+import { amoProdCDN } from './lib/shared';
+
 const staticHost = 'https://addons-admin.cdn.mozilla.net';
 
 module.exports = {
```
Comment (muffinresearch) [inline config/default-admin.js]: It's a bit weird being module.exports and using es6 imports.

Comment (kumar303) [inline config/default-admin.js]: Hmm, I don't think there's an easy way to convert all of these to es6 without turning each object key into `export const ...`

**Comment by kumar303** [comment]
Thanks. Let's try it on dev...

**Pull request merged**